CHANGES.md

```diff
@@ -1,5 +1,18 @@
 # Changes
 
+## unreleased
+
+### Feature
+
+- version updates:
+  - update gradle from 8.5 to 8.7
+  - update kotlin from 1.9.21 to 1.9.23
+  - update detekt from 1.23.4 to 1.23.6
+  - update dokka from 1.9.10 to 1.9.20
+  - update jooq from 3.18.6 to 3.19.6
```

Comment:
PR mainly created for updating the jooq version to be in sync with the one used in `lovely-micronaut-graphql`